### PR TITLE
MOU53 & Spearhead

### PR DESCRIPTION
**Pull request by FungiFellow**
## About the PR

Ported 2 Break action shotguns and made a base for break action weapons. The Spearhead is renamed to the M357 to avoid copyright
## Why / Balance

Parity.
MOU53 is balanced by low ammo count and not having access to buckshot. While the Double Barrel is CLF ONLY

Resolves https://github.com/RMC-14/RMC-14/issues/1446 & https://github.com/RMC-14/RMC-14/issues/1131
## Media

image
image
image
image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added the MOU53 Shotgun & the Double Barrel

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml [hunk 1]
```diff
@@ -74,6 +79,48 @@
       recoilFlat: 2
       scatterFlat: 6
 
+- type: entity
+  parent: RMCStockAttachmentBase
+  id: RMCAttachmentMOU53Stock
+  name: MOU53 Stock
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml:85]:
```suggestion
  name: MOU53 tactical stock
```

File: Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml [hunk 2]
```diff
@@ -74,6 +79,48 @@
...
+        wieldedOnly: true
+      accuracyAddMult: 0.3
+      recoilFlat: -4
+      scatterFlat: -5
+    - conditions:
+        unwieldedOnly: true
+      accuracyMovementPenaltyAddMult: -1
+      accuracyAddMult: -0.1
+      recoilFlat: 2
+      scatterFlat: 5
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml:122]: These modifiers aren't correct, scatter wielded is -1 in 13 which should be -2 here, check AttachableData.cs file for conversions

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml (hunk 2 above)
Comment (FungiFellow) [reply Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml]: Adjusted scatter values

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**